presto-orc/src/main/java/com/facebook/presto/orc/writer/DictionaryColumnWriter.java

```diff
@@ -414,7 +414,7 @@ public long getRetainedBytes()
                 dataStream.getRetainedBytes() +
                 presentStream.getRetainedBytes() +
                 getRetainedDictionaryBytes() +
-                columnStatisticsRetainedSizeInBytes;
+                rowGroupRetainedSizeInBytes;
```

Comment:
nit: It might also make sense to include the size of the `rowGroups`. e.g.: `sizeOfObjectArray(rowGroups.size()) + `